sources/packages/api/src/route-controllers/restriction/restriction.aest.controller.ts

```diff
@@ -126,13 +127,14 @@ export class RestrictionController extends BaseController {
    * @param studentRestrictionId
    * @returns Student restriction detail view.
    */
-  @Groups(UserGroups.AESTUser)
-  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @ApiNotFoundResponse({
+    description: "The student restriction does not exist.",
+  })
   @Get("/student/:studentId/studentRestriction/:studentRestrictionId")
   async getStudentRestrictionDetail(
     @Param("studentId") studentId: number,
     @Param("studentRestrictionId") studentRestrictionId: number,
-  ): Promise<RestrictionDetailDTO> {
+  ): Promise<RestrictionDetailAPIOutDTO> {
```

Comment:
getIDIRUserFullName is used only inside this controller. Once it is changed to getUserFullName, getIDIRUserFullName can be removed.